web/src/App.less

```diff
@@ -1,6 +1,30 @@
 /* stylelint-disable at-rule-name-case */
 /* stylelint-disable selector-class-pattern */
 
+:root {
```

Comment:
maybe this part is not necessary, because ant uses design tokens. I will think about removing this vars